.github/workflows/check-pull-request.yml

```diff
@@ -116,7 +94,7 @@ jobs:
     steps:
       - uses: gsactions/commit-message-checker@v2
         with:
-          pattern: '(?:feat|fix|chore|test|docs|refactor|revert|hotfix)(?:\(\w+\))?:\s[a-z]{1,2}.+(?:\n\nBREAKING CHANGE: .*)?'
+          pattern: '(?:feat|fix|chore|test|docs|refactor|revert|hotfix)(?:\(\w+\))?:\s[a-z]{1,2}.+(?:\n\nBREAKING CHANGE: .*)?|Merge branch .+'
```

Comment:
Also, I'd like you add `ci|style` in the pattern. Remember that `feat` convention is related a functionality for the project and use in that is not a good practice. However, you could use `chore` in that case, which is more related.

Reply:
In the wiki, in the commits section, the `ci|style` is not mentioned, that is why I did not enter it, but you have zaron when it comes to adding a feat, I would have to enter a chore, that is the best, thank you very much for the recommendation.
If we would like to increase the `ci|style` please let me know if we would have to modify the methodologies.